examples/example_tissuemask.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<p>may be change the icon as John did in his 0 to AI notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>